### PR TITLE
Hotfix for missing config attribute issue.

### LINKED ISSUE #5757: Error 500 on source interface and journalist interface after 1.7.0 upgrade
## Description

Initially reported by forum user jonas.franzen in https://forum.securedrop.org/t/issues-after-update-to-1-7-0/1391

Some due to a missing variable in config.py
```
Jan 27 12:48:07 applicationserver python[1136]: AttributeError: module ‘config’ has no attribute ‘SESSION_EXPIRATION_MINUTES’
```

## Steps to reproduce

TBD

## Comments
Forum user dachary reports that simply adding the value to config.py resolves

### PR DESCRIPTION
**Pull request by zenmonkeykstop**
## Status

Ready for review

## Description of Changes

Fixes #5757.

Hotfix PR for missing config attribute issue - contains both the backported fix from #5759 and version/changelog updates (to save CI wall time).

## Testing

- [ ] base is `release/1.7.1`
- [ ] PR contains commit from #5759 
- [ ] PR contains valid version update to 1.7.1 along with associated changelog updates
- [ ] CI passes
- [ ] test plan from #5757 passes. 

### If you made changes to the server application code:

- [x] Linting (`make lint`) and tests (`make test`) pass in the development container

### If you made changes to `securedrop-admin`:

- [ ] Linting and tests (`make -C admin test`) pass in the admin development container

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you added or removed a file deployed with the application:

- [ ] I have updated AppArmor rules to include the change

### If you made non-trivial code changes:

- [x] I have written a test plan and validated it for this PR

Choose one of the following:

- [ ] I have opened a PR in the [docs repo](https://github.com/freedomofpress/securedrop-docs) for these changes, or will do so later
- [ ] I would appreciate help with the documentation
- [ ] These changes do not require documentation

### If you added or updated a code dependency:

Choose one of the following:

- [ ] I have performed a diff review and pasted the contents to [the packaging wiki](https://github.com/freedomofpress/securedrop-debian-packaging/wiki)
- [ ] I would like someone else to do the diff review

### CODE REVIEW

**Review by conorsch — APPROVED** [review]
Looks good based on visual review.

**Comment by conorsch** [comment]
Associated docs PR is here: https://github.com/freedomofpress/securedrop-docs/pull/142

File: install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/changelog-focal [hunk 1]
```diff
@@ -1,3 +1,9 @@
+securedrop-app-code (1.7.1+focal) focal; urgency=medium
```
Comment (emkll) [inline install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/changelog-focal:1]: We will need to clean this up in 1.8.0, as we didn't release 1.7.1 for focal

**Comment by conorsch** [comment]
The test failures in admin-tests here: https://app.circleci.com/pipelines/github/freedomofpress/securedrop/1760/workflows/46441d05-f6f1-45f4-8b90-24a72c05a54a/jobs/49540 appear to be related to the fact that we're bumping the version and creating a tag in a single pass, as part of this release. @zenmonkeykstop Does that sound right to you? If so, I suggest we disregard that particular failure and press on.

**Review by emkll — APPROVED** [review]
I think this is good to merge in the 1.7.1 release branch, these changes have been already tagged and released. Only deb-tests-focal is failing (deb-tests-focal  and out of date builder)

**Comment by conorsch** [comment]
Agreed. The Focal-only test failures are consistent with what we saw in https://github.com/freedomofpress/securedrop/pull/5753#issuecomment-767883364, so proceeding with merge.

**Pull request merged**